src/ORM/Connect/TempDatabase.php

```diff
@@ -183,6 +229,11 @@ public function deleteAll()
      */
     public function resetDBSchema(array $extraDataObjects = [])
     {
+        // pgsql doesn't allow schema updates inside transactions
```

Comment:
TBH, I think this is most DBMS as in MySQL alter tables cause an implicit commit for transactions too...